cfripper/rules/SQSQueuePolicyWildcardActionRule.py

```diff
@@ -21,7 +21,7 @@ class SQSQueuePolicyWildcardActionRule(Rule):
 
     REASON = "SQS Queue policy {} should not allow * action"
 
-    def invoke(self, resources, parameters):
-        for resource in resources.get("AWS::SQS::QueuePolicy", []):
-            if resource.policy_document.wildcard_allowed_actions(pattern=r"^(\w*:){0,1}\*$"):
+    def invoke(self, cfmodel):
+        for resource in cfmodel.resources.get("AWS::SQS::QueuePolicy", []):
+            if resource.policy_document.allowed_actions_with(re.compile(r"^(\w*:){0,1}\*$")):
```

Comment:
Extract the regex into a constant, it doesn't make sense to compile the regex every single iteration of the loop